ShopifySharp/Entities/SalesChannel/PaymentSalesChannel.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace ShopifySharp.Entities.SalesChannel
+{
+    public class PaymentSalesChannel
+    {
+        [JsonProperty("id")]
+        public long Id { get; set; }
+
+        [JsonProperty("unique_token")]
+        public string UniqueToken { get; set; }
+
+        [JsonProperty("payment_processing_error_message")]
+        public object PaymentProcessingErrorMessage { get; set; }
+
+        [JsonProperty("next_action")]
+        public NextAction NextAction { get; set; }
+
+        [JsonProperty("fraudulent")]
+        public bool Fraudulent { get; set; }
+
+        [JsonProperty("transaction")]
+        public Transaction Transaction { get; set; }
+
+        [JsonProperty("credit_card")]
+        public PaymentCreditCard CreditCard { get; set; }
+
+        [JsonProperty("checkout")]
+        public CheckoutSalesChannel Checkout { get; set; }
+    }
+
+    public class PaymentCreditCard
```

Comment:
The extra classes in this file should be moved into their own file.

Reply:
Done